RInAppMessaging/Classes/Models/Responses/GetConfigResponse.swift

```diff
@@ -3,6 +3,7 @@ internal struct GetConfigResponse: Decodable {
 }
 
 internal struct ConfigData: Decodable {
+    let rolloutPercentage: Int
     let enabled: Bool
```

Comment:
`enabled` will be removed in an other ticket.

Reply:
OK